admob/src_java/com/google/firebase/admob/internal/cpp/ConstantsHelper.java

```diff
@@ -100,14 +100,14 @@
   public static final int AD_VIEW_PRESENTATION_STATE_OPENED_PARTIAL_OVERLAY = 3;
 
   public static final int AD_VIEW_PRESENTATION_STATE_COVERING_UI = 4;
-  // LINT.ThenChange(//depot_firebase_cpp/admob/client/cpp/src/include/firebase/admob/banner_view.h,
-  // //depot_firebase_cpp/admob/client/cpp/src/include/firebase/admob/native_express_ad_view.h)
-
+  // LINT.ThenChange(//depot_firebase_cpp/admob/client/cpp/src/include/firebase/admob/banner_view.h)
```

Comment:
I don't think we still have IfThisThenThat, so it might be cleaner to get rid of these. Unless there is a GitHub check equivalent - that would be awesome.

Reply:
Ok. Yeah, I'm actually removing some of the stuff in here in the next PR, so I'll remove those, too.  I didn't know this was related to IFTTT!